ERROR: Traceback (most recent call last):
  File "H:\GIS_Data\Work\NRCan\FGP\TA001\FGP_WebExtractor\scripts\BC_extractor.py", line 362, in extract_catalogue
    ds_res = ds_json['result']
TypeError: 'NoneType' object has no attribute '__getitem__'


null